emc-metalnx-services/src/main/java/com/emc/metalnx/services/configuration/ConfigServiceImpl.java

```diff
@@ -440,12 +440,12 @@ public void setIrodsZone(String irodsZone) {
 
 	@Override
 	public void setIrodsJobUser(String irodsJobUser) {
```

Comment:
This is still referencing the "job user". Is that... okay?

Reply:
I thought I changed those.

Reply:
I updated this.